activiti-cloud-starter-audit/src/main/java/org/activiti/cloud/starter/audit/configuration/EnableActivitiAudit.java

```diff
@@ -1,4 +1,4 @@
-package org.activiti.cloud.starter.configuration;
+package org.activiti.cloud.starter.audit.configuration;
```

Comment:
Is the intention to remove `@DiscoveryClient ` here too?
I'm also thinking we shouldn't be using EnableActivitiAudit on the spring boot app in activiti-cloud-audit as this annotation effectively bypasses any `@ConditionalOnProperty` settings we put (or should put) on the AutoConfiguration